### ISSUE TITLE
Add Set All button in Send Credit dialog to fill in all available balance

### ISSUE THREAD

**Issue by kovalur** [lbryio/lbry-desktop#3417]
**Is your feature request related to a problem? Please describe.**
From to time, I'm sending all available balance between wallets, and I should manually to type in this amount.

**Describe the solution you'd like**
Would be nice to have (maybe right in the field) a button to set all available balance.

**Comment by tzarebczan** [comment]
Thanks for filing the issue! This is currently blocked by https://github.com/lbryio/lbry-sdk/issues/1628
